pkg/apps/util/util.go

```diff
@@ -782,6 +788,23 @@ func DeploymentsForCleanup(configuration *deployapi.DeploymentConfig, deployment
 	return relevantDeployments
 }
 
+func ConfigExceededTimeoutSeconds(config *deployapi.DeploymentConfig, latestRC *v1.ReplicationController) bool {
```

Comment:
DeploymentExceededTimeoutSeconds

Reply:
right, I was lazy to change it, now you beat me to it :)